spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/connection/ConnectionEventTests.java

```diff
@@ -127,6 +140,138 @@ public void testNioServerExceptionEvent() throws Exception {
 		testServerExceptionGuts(port, factory);
 	}
 
+	@Test
+	public void testOCANoConnectionEvents() {
```

Comment:
I think we should provide the fully qualified names for test methods anyway. At least it isn't so long in this case